### DISCUSSION TITLE
Does microrealestate enable tenant-side payments?

### DISCUSSION THREAD

**Discussion by aram0112** [microrealestate/microrealestate#181] [General]
Hi! Thank you for this project. I deployed microrealestate and followed the steps to create a contract, property and tenant. Upon signing in to the tenant's portal I don't see any functionality to interact with the existing lease in anyway or initiate any payments. Is this beyond the scope of the project currently or am I missing any steps? Thank you so much!

**Comment by camelaissani** [comment]
Hi @aram0112 ! Currently the tenant portal allowed the tenants to download their invoices and see some details of their contracts. No electronic payments can be initiated from there so far.

**Comment by z4cth3p4c** [comment]
Would love to see tenants have the ability to pay their balance. Are there any plans to add this?

**Comment by camelaissani** [comment]
No plan so far but there is an open issue on this topic:

https://github.com/microrealestate/microrealestate/issues/117
